Gateway/Response/CaptureHandler.php

```diff
@@ -24,13 +24,15 @@ class CaptureHandler extends AbstractResponseHandler implements HandlerInterface
      */
     public function handle(array $handlingSubject, array $response)
     {
-        if (!isset($handlingSubject['payment'])
+        if (
+            !isset($handlingSubject['payment'])
             || !$handlingSubject['payment'] instanceof PaymentDataObjectInterface
         ) {
             throw new \InvalidArgumentException('Payment data object should be provided');
         }
 
-        if (!isset($response['object'])
+        if (
+            !isset($response['object'])
             || !$response['object'] instanceof TransactionResponse
         ) {
             throw new \InvalidArgumentException('Data must be an instance of "TransactionResponse"');
```

Comment:
these checks are in multiple files, maybe refactor them into a abstract class/ trait

Reply:
Thank you, I created this class Buckaroo\Magento2\Gateway\Helper\SubjectReader. I will update the code to be used